source.txt

```diff
@@ -204,18 +204,19 @@ Table of Contents
          }
        }
 
-    All folders are treated as existing, and therefore GET requests to
-    untouched folders SHOULD be responded to with a folder description
-    with no items (the items field set to '{}'). However, an empty
-    folder MUST NOT be listed as an item in its parent folder.
-
-    Also, since folders exist automatically, PUT and DELETE requests
-    only need to be made to documents, and never to folders. A document
-    PUT will make all ancestor folders along its path become non-empty;
-    deleting the last document from a subtree will make that whole
-    subtree become empty. Folders will therefore show up in their parent
-    folder descriptions if and only if their subtree contains at least
-    one document.
+    GET requests to empty folders SHOULD be responded to with a folder
+    description with no items (the items field set to '{}'). However, an
+    empty folder MUST NOT be listed as an item in its parent folder.
+
+    PUT and DELETE requests only need to be made to documents, and never
+    to folders. A document PUT will make all ancestor folders along its
+    path become non-empty; deleting the last document from a subtree
+    will make that whole subtree become empty. Folders will therefore
+    show up in their parent folder descriptions if and only if their
+    subtree contains at least one document.
+
+    In contexts outside of this document, non-empty folders may be
+    called 'existent', while empty folders may be called 'non-existent'.
```

Comment:
I thought about adding something like:

>However, this terminology is not really applicable here, since GET requests on empty folders do not return a 404.

I'm not sure how to fully explain that reasoning.